### PR TITLE
Translate README.md to Chinese

### PR DESCRIPTION
**Pull request by jercylew**
## Motivation

To add Chinese version document of README.md into this project

## Modification

The corresponding Chinese version of the document [README_zh-CN.md](https://github.com/open-mmlab/mmaction2/blob/main/README_zh-CN.md) was updated.

## BC-breaking (Optional)

No!

## Use cases (Optional)

NA

## Checklist

1. Pre-commit or other linting tools should be used to fix the potential lint issues.
2. The modification should be covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation should be modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -2,35 +2,35 @@
   <img src="https://github.com/open-mmlab/mmaction2/raw/main/resources/mmaction2_logo.png" width="600"/>
   <div>&nbsp;</div>
   <div align="center">
-    <b><font size="5">OpenMMLab 官网</font></b>
+    <b><font size="5">OpenMMLab官网</font></b>
```
Comment (ly015) [inline README_zh-CN.md:5]: Thanks for your help. This is actually one of our conventions that there should be a whitespace between English and Chinese characters. Therefore there is no need for any modifications.

File: README_zh-CN.md [hunk 2]
```diff
@@ -2,35 +2,35 @@
...
-[👀模型库](https://mmaction2.readthedocs.io/zh_CN/latest/modelzoo.html) |
-[🆕更新](https://mmaction2.readthedocs.io/zh_CN/latest/notes/changelog.html) |
-[🚀进行中项目](https://github.com/open-mmlab/mmaction2/projects) |
-[🤔问题反馈](https://github.com/open-mmlab/mmaction2/issues/new/choose)
+[📘文档](https://mmaction2.readthedocs.io/en/latest/) |
+[🛠️安装](https://mmaction2.readthedocs.io/en/latest/get_started/installation.html) |
+[👀模型库](https://mmaction2.readthedocs.io/en/latest/model_zoo/modelzoo.html) |
+[🆕更新日志](https://mmaction2.readthedocs.io/en/latest/notes/changelog.html) |
+[🚀正在进行的项目](https://github.com/open-mmlab/mmaction2/projects) |
+[🤔报告问题](https://github.com/open-mmlab/mmaction2/issues/new/choose)
```
Comment (ly015) [inline README_zh-CN.md:33]: We would like to keep these links to Chinese documentation.

File: README_zh-CN.md [hunk 3]
```diff
@@ -290,33 +316,32 @@ MMAction2 将跟进学界的最新进展，并支持更多算法和框架。如
...
+- [MMOCR](https://github.com/open-mmlab/mmocr): OpenMMLab文本检测，识别及理解工具包
+- [MMPose](https://github.com/open-mmlab/mmpose): OpenMMLab姿态预测工具包及基准
+- [MMHuman3D](https://github.com/open-mmlab/mmhuman3d): OpenMMLab 3D人物参数模型工具包及基准
+- [MMFewShot](https://github.com/open-mmlab/mmfewshot): OpenMMLab少样本学习工具包及基准
+- [MMAction2](https://github.com/open-mmlab/mmaction2): OpenMMLab下一代动作理解工具并及基准
+- [MMFlow](https://github.com/open-mmlab/mmflow): OpenMMLab光流法工具并及基准
+- [MMDeploy](https://github.com/open-mmlab/mmdeploy): OpenMMLab模型部署框架
+- [MMRazor](https://github.com/open-mmlab/mmrazor): OpenMMLab模型压缩工具包及基准
+- [MIM](https://github.com/open-mmlab/mim): MIM用于安装OpenMMLab所需包
+- [Playground](https://github.com/open-mmlab/playground): 一个用于收集和展示基于OpenMMLab构建的优质项目中心站点
```
Comment (ly015) [inline README_zh-CN.md:347]: These descriptions are used in all repos in a unified form. So we prefer not to change them in MMAction2 if there is no special reason.

File: README_zh-CN.md (hunk 1 above)
Comment (jercylew) [reply README_zh-CN.md]: Updated. add space back between the two

File: README_zh-CN.md (hunk 2 above)
Comment (jercylew) [reply README_zh-CN.md]: Links updated to Chinese version

File: README_zh-CN.md (hunk 3 above)
Comment (jercylew) [reply README_zh-CN.md]: Okay, I restore them to the original ones

**Review by cir7 — APPROVED** [review]

**Pull request merged**